src/frontend/src/flows/authorize/index.ts

```diff
@@ -71,13 +72,13 @@ export const authnTemplateAuthorize = ({
     <div class="c-origin-preview c-origin-preview--header">
       <img
         class="c-origin-preview__logo c-origin-preview__logo--background"
-        src=${logo}
+        src=${BASE_URL + "icons/" + logo}
```

Comment:
The `BASE_URL + "icons/"` pattern is used in a couple of places, and ideally should be defined only once. I'll tackle this when adding support for preloading those images.

Reply:
https://github.com/dfinity/internet-identity/pull/1678